dataset_torch.py

```diff
@@ -0,0 +1,109 @@
+#%% Imports
+import torch
```

Comment:
I'm sure I'm missing something, but what's the difference between `code/dataset.py` and `dataset_torch.py`?

Reply:
dataset does not depend on torch. I was thinking this is the platform independent files (e.g. not tensorflow or pytorch), and that you only needed pytorch if you use dataset_torch. Not sure if this is too much of a modularization...

Reply:
`dataset.py` imports from pytorch too though? `from torch.utils.data import Dataset, DataLoader`
I think it's a good idea to at some point make it accessible for tensorflow or other frameworks as well, but it doesn't seem like that's achieved now?

I think making a generator of some sort exposing each userslate in numpy format is probably the most easily extensible in that case? I think it's smart to avoid importing both TensorFlow and Pytorch in this package before exporting

Reply:
dataset.py doesnt exist anymore (git pull), but datahelper.py does. The only thing it does currently is downloading the data tho.